static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -110,8 +111,24 @@ const Track = styled('div')({
     top: '-4px',
 });
 
-const Wave = styled('svg')({
+const FakeWave = styled('div')({
     flex: 1,
+    svg: {
+        width: '100%',
+        transform: `translate(0px, 11px)`,
```

Comment:
Let's see if wrapping in a `DIV` helps you get rid of those transforms

Reply:
hmm it didn't seem to, but will investigate

Reply:
forget it 👋 it's a test, we can polish later

Reply:
ha cool.